src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2489,6 +2490,7 @@ private function registerNotifierConfiguration(array $config, ContainerBuilder $
             $container->getDefinition('chatter.transports')->setArgument(0, $config['chatter_transports']);
         } else {
             $container->removeDefinition('chatter');
+            $container->removeAlias(ChatterInterface::class);
         }
         if ($config['texter_transports']) {
             $container->getDefinition('texter.transports')->setArgument(0, $config['texter_transports']);
```

Comment:
Do we need to do the same here for the texter transports?

Reply:
I think so i get the same error for texter

Reply:
indeed, done